data/strings/strings.txt

```diff
@@ -25419,6 +25419,16 @@
     zh-Hans = 该路线将改为汽车（路线）
     zh-Hant = 該路線將改為汽車（路線）
 
+  [not_all_shown_bookmarks_carplay]
+    tags = ios
+    en = Not all bookmarks are shown
```

Comment:
Slightly corrected output of `tools/python/translate.py Not all bookmarks are shown`

P.S. What's the difference of `angezeigt` vs `zu sehen` in this context?

```
    en = Not all bookmarks are shown
    af = Nie alle boekmerke word gewys nie
    ar = لا يتم عرض جميع الإشارات المرجعية
    az = Bütün əlfəcinlər göstərilmir
    be = Не ўсе закладкі паказаны
    bg = Не всички отметки се показват
    ca = No es mostren totes les adreces d'interès
    cs = Nezobrazují se všechny záložky
    da = Ikke alle bogmærker vises
    de = Es werden nicht alle Lesezeichen zu sehen
    el = Δεν εμφανίζονται όλοι οι σελιδοδείκτες
    es = No se muestran todos los marcadores
    et = Kõiki järjehoidjaid ei kuvata
    eu = Ez dira laster-marka guztiak erakusten
    fa = همه نشانک ها نشان داده نمی شوند
    fi = Kaikkia kirjanmerkkejä ei näytetä
    fr = Tous les signets ne sont pas affichés
    he = לא כל הסימניות מוצגות
    hi = सभी बुकमार्क नहीं दिखाए गए हैं
    hu = Nem minden könyvjelző jelenik meg
    id = Tidak semua penanda ditampilkan
    it = Non tutti i segnalibri vengono visualizzati
    ja = すべてのブックマークが表示されるわけではない
    ko = 모든 북마크가 표시되는 것은 아닙니다.
    lt = Rodomos ne visos žymės
    mr = सर्व बुकमार्क दर्शविले जात नाहीत
    nb = Ikke alle bokmerker vises
    nl = Niet alle bladwijzers worden getoond
    pl = Nie wszystkie zakładki są wyświetlane
    pt = Nem todos os marcadores são mostrados
    pt-BR = Nem todos os marcadores são exibidos
    ro = Nu sunt afișate toate marcajele
    ru = Отображаются не все метки
    sk = Nezobrazujú sa všetky záložky
    sv = Alla bokmärken visas inte
    sw = Sio vialamisho vyote vinavyoonyeshwa
    th = บุ๊กมาร์กไม่แสดงทั้งหมด
    tr = Tüm yer imleri gösterilmez
    uk = Відображаються не всі мітки
    vi = Không phải tất cả dấu trang đều được hiển thị
    zh-Hans = 并非所有书签都会显示
    zh-Hant = 未顯示所有書籤
```

Reply:
Ah thanks I didn't know that script. I added the missing translations.

> P.S. What's the difference of angezeigt vs zu sehen in this context?

I think `Es werden nicht alle Lesezeichen zu sehen` is grammatically wrong but I can't tell you why. You could say `Es sind nicht alle Lesezeichen zu sehen` but if something is shown on a screen `angezeigt` sounds better. 

Take this with a grain of salt because I'm Swiss, so my German is kind of messed up 😅